lib/brexit_checker/groups.rb

```diff
@@ -0,0 +1,28 @@
+class BrexitChecker::Groups
+  include ActiveModel::Validations
+
+  GROUPS_PATH = Rails.root.join("lib", "brexit_checker", "groups.yaml")
+
+  validates_presence_of :key, :text, :priority
+  validates_inclusion_of :key, in: %w(visiting-eu
+                                      visiting-uk
+                                      visiting-ie
+                                      living-eu
+                                      living-ie
+                                      living-uk
+                                      working-uk
+                                      studying-eu
+                                      studying-uk)
+
+  attr_reader :key, :text, :priority
+
+  def initialize(attrs)
```

Comment:
I'm wondering if it makes sense to store the groups as a hash of ```key->{text, priority}```

Reply:
Lets run through this?